stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/plugin/state/global/GlobalState.kt

```diff
@@ -105,31 +135,82 @@ public interface GlobalState {
      *
      * @return True if the user is online otherwise False.
      */
+    @Deprecated(
+        message = "Use ClientState.isOnline() instead.",
+        replaceWith = ReplaceWith(
+            expression = "ChatClient.instance().clientState.isOnline",
+            imports = [
+                "io.getstream.chat.android.client.ChatClient",
+                "io.getstream.chat.android.offline.extensions.clientState",
+            ]
+        )
+    )
     public fun isOnline(): Boolean
 
     /**
      * If the user is offline or not.
      *
      * @return True if the user is offline otherwise False.
      */
+    @Deprecated(
+        message = "Use ClientState.isOffline() instead.",
+        replaceWith = ReplaceWith(
+            expression = "ChatClient.instance().clientState.isOffline",
+            imports = [
+                "io.getstream.chat.android.client.ChatClient",
+                "io.getstream.chat.android.offline.extensions.clientState",
+            ]
+        )
+    )
     public fun isOffline(): Boolean
 
     /**
      * If connection is in connecting state.
      *
      * @return True if the connection is in connecting state.
      */
+    @Deprecated(
+        message = "Use ClientState.isConnecting() instead.",
+        replaceWith = ReplaceWith(
+            expression = "ChatClient.instance().clientState.isConnecting",
+            imports = [
+                "io.getstream.chat.android.client.ChatClient",
+                "io.getstream.chat.android.offline.extensions.clientState",
+            ]
+        )
+    )
     public fun isConnecting(): Boolean
 
     /**
      * If domain state is initialized or not.
      *
      * @return True if initialized otherwise False.
      */
+    @Deprecated(
+        message = "Use ClientState.isInitialized() instead.",
+        replaceWith = ReplaceWith(
+            expression = "ChatClient.instance().clientState.isInitialized",
+            imports = [
+                "io.getstream.chat.android.client.ChatClient",
+                "io.getstream.chat.android.offline.extensions.clientState",
+            ]
+        )
+    )
     public fun isInitialized(): Boolean
 
     /**
      * Clears the state of [GlobalState].
      */
+    @Deprecated(
+        message = "Use ClientMutableState.clearState()",
+        replaceWith = ReplaceWith(
+            expression = "ChatClient.instance().clientState.toMutableState()?.clearState()",
+            imports = [
+                "io.getstream.chat.android.client.ChatClient",
+                "io.getstream.chat.android.offline.extensions.clientState",
+                "io.getstream.chat.android.client.setup.state.internal.toMutableState",
+            ]
+        )
+    )
     public fun clearState()
```

Comment:
Why do we deprecate `clearState`? 🤔 Don't we need to clear `GlobalState` related things like `totalUnreadCount`?

Reply:
🤦 You're right, let me remove this.